python/modules/Stop0l_trigger.py

```diff
@@ -235,22 +233,30 @@ def analyze(self, event):
 			zmumu_mid.append(zmumu_cand)
 	n_zmumu = len(zmumu_mid)
 
-	photon_loose = []
-	for photon in photons:
-		if (self.SelPhotons(photon)):
-			photon_loose.append(photon)
-	n_photon = len(photon_loose)
+        photon_loose = []
+        photon_mid = []
+        for photon in photons:
+                if (abs(photon.eta) < 1.442 or (1.566 < abs(photon.eta) and abs(photon.eta) < 2.5)):
+                        cutbase =  photon.cutBasedBitmap if self.era != "2016" else photon.cutBased
+                        if(cutbase >=1):
```

Comment:
I think Caleb found an issue with photon ID. Can you cross check with him?

Reply:
I would recommend doing it this way instead: 
https://github.com/susy2015/NanoSUSY-tools/blob/master/python/modules/Stop0lObjectsProducer.py#L147-L153

Reply:
It is better to be explicit with using photon.cutBased for 2016 and photon.cutBasedBitmap for 2017 and 2018, where photon.cutBasedBitmap is actually treated like a bit map.

Reply:
This will also make our use of photon ID consistent.

Reply:
Our expressions are identical. Loose ID is 1 and medium ID is 2 for both int and bit. But I can change to your expression if you recommend.

Reply:
But remember that the ID values are not the same in 2016 and 2017,2018.
```
2016: (0:fail, 1: :loose, 2:medium, 3:tight)
2017,2018: (0:fail, 1: :loose, 3:medium, 7:tight)
```

Reply:
That is why using bit logic is better for 2017, 2018 since the ID is a bitmap for those years.

Reply:
So you method does not work for tight ID because the value of tight ID is 3 in 2016 and 7 in 2017,2018.

Reply:
OK. I have changed to your expression. How to include my new commit
https://github.com/susy2015/NanoSUSY-tools/commit/bbc6819f6f4e7c81ef6593100ef447aa50a46f5d
into this PR?

Reply:
So we are good to go?

Reply:
I tested locally and got identical photon trigger eff as my old expression. So I think we are good to go

Reply:
ok

Reply:
Thanks @wang-hui .

Reply:
@wang-hui also, I noticed that you need the updated eta cut: https://github.com/susy2015/NanoSUSY-tools/blob/master/python/modules/Stop0lObjectsProducer.py#L143-L144

Reply:
Specifically please use 

```if (abeta > 1.4442 and abeta < 1.5660) or (abeta > 2.5):```